sleep_posix.cc

```diff
@@ -0,0 +1,33 @@
+#include "sleep.h"
+
+#if ( _POSIX_C_SOURCE >= 200112L ) && ( !NAN_HAS_CPLUSPLUS_11 )
+
+#include <unistd.h>
+#include <sys/time.h>
+
+void node_sleep(unsigned seconds) {
```

Comment:
No problem for late review, I guess I had to proactively check with you by other means, no wait for review indefinitely. Will implement suggested changes in few days.
I implemented that way because I believe sleep() uses less processor power (i.e. processor is in fact sleeping more instead of calculating remaining microseconds). But this module is used mainly for debugging anyway, is not useful in production and interrupting early is more user-friendly, so will do according to your comment.

Reply:
Now node_usleep is the only function implemented in native code, sleep and msleep are implemented in Javascript (index.js)